scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/DefaultModelJobValidatorServiceProviderTest.java

```diff
@@ -155,6 +160,17 @@ private TaskFlowJob createJobWithSpelJobModelVariablesOK() throws UserException
         return job;
     }
 
+    private TaskFlowJob createJobWithSpelJobModelVariablesUnauthorizedType() throws UserException {
+        TaskFlowJob job = new TaskFlowJob();
+        job.setVariables(ImmutableMap.of("var1",
+                                         new JobVariable("var1",
+                                                         "value1",
+                                                         SPEL_LEFT +
+                                                                   "T(Runtime).exec('hostname').waitFor() instanceof T(Integer)" +
```

Comment:
actually there was a mistake in this string, I should have used T(java.lang.Runtime).getRuntime().exec( ..